### PR TITLE
NDI-28: Styled campaign conduct survey pages

### PR DESCRIPTION
**Pull request by mukeshcompucorp**
# Description
This PR
- Fixes minor glitches of data tables
- Styles conduct survey pages

# Screenshots
**Before:**
before-1
**After:**
after-1

**Before:**
before-2
**After:**
after

# Tests
- BackstopJS tested

### CODE REVIEW

**Comment by AkA84** [comment]
@mukeshcompucorp i can't review this PR, there is code that belongs to this PR https://github.com/civicrm/org.civicrm.shoreditch/pull/127/files, not sure what's going on

**Comment by AkA84** [comment]
@mukeshcompucorp also the buttons in the screenshot look really ugly (dark and all clumped together)

**Comment by mukeshcompucorp** [comment]
> @mukeshcompucorp i can't review this PR, there is code that belongs to this PR https://github.com/civicrm/org.civicrm.shoreditch/pull/127/files, not sure what's going on

This was because NDI-23 was not merged. Now it's fine.

> @mukeshcompucorp also the buttons in the screenshot look really ugly (dark and all clumped together)

This is done and screenshot updated

File: scss/civicrm/campaign/pages/_campaign.scss [hunk 1]
```diff
@@ -17,4 +17,62 @@
...
+    }
+
+    .crm-copy-fields {
+      border-bottom: 1px solid $gray-light;
+      border-top: 1px solid $gray-light;
+    }
+
+    .civireport-criteria {
+      .form-layout-compressed {
+        @extend %crm-table;
```
Comment (AkA84) [inline scss/civicrm/campaign/pages/_campaign.scss]: is this `@extend` really necessary?

Comment (mukeshcompucorp) [reply scss/civicrm/campaign/pages/_campaign.scss]: yes, or I need to write all css for that table again. with targeting each part. so, better make it extended.

**Pull request merged**